client/src/components/v2/CameraRecording.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {
+  Button, Card, Col, Row,
+} from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import { startRecording, stopRecording, getPrettyDeviceName } from 'api/v2/camera';
+import CameraRecordingStatus from './CameraRecordingStatus';
+import styles from './CameraRecording.module.css';
+
+/**
+ * @typedef {object} CameraRecordingProps
+ * @property {Array} devices Array of strings of device names
+ */
+
+/**
+ * Camera recording buttons for starting and stopping video recording.
+ *
+ * Starts/stops recording for both displays.
+ *
+ * This is a feature intended for V3 but is currently in V2 for testing.
+ *
+ * @param {CameraRecordingProps} props Props
+ * @returns {React.Component<CameraRecordingProps>} Component
+ */
+export default function CameraRecording({ devices }) {
+  return (
+    <Card>
+      <Card.Body>
+        <Card.Title>Recording Controls</Card.Title>
+        <Row className={styles.topMarginContainer}>
+          {
+            devices.map((device) => (
+              <Col
+                className={styles.topMargin}
+                key={device}
+                sm={Math.floor(12 / devices.length)}
```

Comment:
This can simply be a value of 6. In the case of a new device, it will simply overflow in to a new line. If you want all the devices to fit on exactly 1 line, this can be implemented with a CSS flexbox.

Reply:
Done
924f560